src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid $color-grey-accent;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+    margin-top: 40px;
+  }
+
+  &__description-title {
+    @extend %description-title-style;
+    color: $color-main;
+  }
+
+  &__description-title:hover {
+    @extend %description-title-style;
+    color: $color-hover-text;
+    transition: color 0.5s;
+  }
+
+  &__description-code {
+    @extend %description-code-style;
+  }
+
+  &__review {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__review-amount {
+    align-self: center;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-main;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-cost {
+    @extend %price-cost-style;
+    color: $color-main;
+  }
+
+  &__price-name {
+    @extend %price-name-style;
+    color: $color-secondary;
+  }
+
+  &__button {
+    width: 100%;
+    display: flex;
+    margin-top: 16px;
+    text-align: center;
+    justify-content: center;
+  }
+
+  &__button-link {
+    align-items: center;
+    display: flex;
+    color: $color-white-accent;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 38px;
+    text-transform: uppercase;
+    background-color: $color-blue-accent;
+    border-radius: 5px;
+    padding: 0 70px;
+    border: 1px solid $color-blue-accent;
+    text-decoration: none;
+    transition: background-color 0.5s, color 0.5s;
+  }
+
+  &__button-link:hover {
+    align-items: center;
+    display: flex;
+    color: $color-blue-accent;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 38px;
+    text-transform: uppercase;
+    background-color: $color-white-accent;
+    border-radius: 5px;
+    padding: 0 70px;
+    border: 1px solid $color-blue-accent;
+    text-decoration: none;
+    transition: background-color 0.5s, color 0.5s;
+  }
+
+  &--position {
+    margin-right: $gap;
+  }
+
+  &--position:nth-child(-n+4) {
+    margin-bottom: 46px;
+  }
+
+  &--position:nth-child(4n) {
+    margin-right: 0;
+  }
+}
+
+.card:hover {
```

Comment:
This should be moved to the card block:

<img width="501" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/87986955/adb2a1c3-5150-4ab6-8f90-2a3db7c4e23f">

Because the transition was setup on hover, when you un-hover the element transition is removed and animation cut. If you setup transition on the class rather than pseudo-class transition is always there and works both ways.

Reply:
Same for all other transitions in this pr (they end abruptly rather than slowly revert), declare them on regular classes rather than with pseudo-classes :hover